packages/features/webhooks/lib/cron.ts

```diff
@@ -56,33 +71,15 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       headers["X-Cal-Signature-256"] = createWebhookSignature({ secret: webhook.secret, body: job.payload });
     }
 
-    try {
-      await fetch(job.subscriberUrl, {
+    fetchPromises.push(
+      fetch(job.subscriberUrl, {
         method: "POST",
         body: job.payload,
         headers,
-      });
-    } catch (error) {
-      console.log(`Error running webhook trigger (retry count: ${job.retryCount}): ${error}`);
-
-      // if job fails, retry again for 5 times.
-      if (job.retryCount < 5) {
```

Comment:
retry was not implemented correctly and it returned `ok: false` on the first error. This also caused that all the other jobs were not triggered

I don't think we need to retry if the request fails, now we are just login the error and continue with the other jobs